sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamSuite.scala

```diff
@@ -211,6 +217,7 @@ class StreamSuite extends StreamTest {
       CheckSinkLatestBatchId(1),
       // Add some data in batch 2
       AddData(inputData, 7, 8, 9),
+      EnsureManualClockInWaitingState,
```

Comment:
I guess my question is would it be okay to just add this logic into `AdvanceManualClock` instead of adding another state that later developers could forget to call?